sdk/keyvault/azure-security-keyvault-administration/samples/README.md

```diff
@@ -0,0 +1,14 @@
+---
+page_type: sample
+languages:
+  - c++ 14
+products:
+  - azure
+  - azure-key-vault
+name: Azure Security KeyVault Administration samples for C++
+description: Samples for the azure-security-keyVault-administration client library.
+---
+
+# Azure.Security.KeyVault.Secrets Samples
+
+- Get Settings, Get Setting , Update setting value
```

Comment:
Nit: We return a List for GetSettings.  Perhaps we should standardize on `List`Settings when returning multiple items.  No need to change now, but do have the conversation in the arch review